free-3.md

```diff
@@ -4,51 +4,96 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+A DSL to describe video game tactics. This would help newer plays learn from more
+experienced players by providing experienced players a way to share tactical or
+strategic advice.
+
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The need met by this idea is the need to learn how to play games better. I'd be
+helping people who want share their strategies with others and those who want
+to learn.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL could be appropriate because it would provide a way for users to compare
+between different tactics and strategies.  In some games it could also help users
+understand losses by looking up the strategies that were used against them.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+This excites me because gaming is very fun and exciting and it's an activity that
+I love to do.  I think a number of Mudders also like games so it could be an idea
+that people here might think is fun.  Moreover there are many games that I am so
+clueless to the tactics, such as the game Diplomacy, that I can't really play well
+enought to enjoy.  If there was an easy way to learn or read about tactics it would
+increase the number of games people would be interested in playing.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Gaming strategy and tactic descriptions
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+A user would either write a strategy or read a strategy or search for a strategy.
+The reason this could be the right way to interact with the problem domain is that
+it could provide a simplified manner to undestand strategies. Currently in many
+games people will share their thoughts on streams like [Twith](http://www.twitch.tv)
+but there is not an organized collected place to view these strategies.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The program running would be people reading or searching for strategies.  An error
+that could occur is that someone could write a very ineffective strategy and then
```

Comment:
It sounds like a DSL isn't a great solution - instead, a community run effort, similar to perhaps stack overflow  (in fact, [Arqade](http://gaming.stackexchange.com) does this to an extent), where users follow a certain set of guidelines in how to write up guides/tactics in order to build a database of tactics.